setup.cfg

```diff
@@ -75,7 +75,7 @@ extras = True
 # e.g. --cov-report html (or xml) for html/xml output or --junitxml junit.xml
 # in order to write a coverage file that can be read by Jenkins.
 addopts =
-    --cov sympc --cov-report term-missing
+;    --cov sympc --cov-report term-missing
```

Comment:
Add this back

Reply:
Yep 😅 
I usually remove that as it gets in the way of PyCharm's pytest plugin for some reason 😛